src/cli.js

```diff
@@ -27,11 +27,13 @@ var options = {
   decrypt: pessimist.d
 };
 
-start(options).then(function() {
-  var timeTaken = ((Date.now() - startTime) / 1000).toFixed(2);
-  console.log('Operation completed successfully in', timeTaken, 'seconds.');
-  process.exit(0);
-}).catch(function(error) {
-  console.error('ERROR', error);
-  process.exit(1);
+start(options, function(data) {
```

Comment:
so start is passed a callback which will be passed a promise?

Reply:
AH wrong file! I'll fix that.

Reply:
since WriteData uses bluebird for promises, I think we use it's [`asCallback`](http://bluebirdjs.com/docs/api/ascallback.html) method to have it work as either one. That way, WriteData internally can use promises but its external interface can be node style callbacks.